src/main/java/com/hivemq/cli/commands/shell/ShellCommand.java

```diff
@@ -37,60 +40,59 @@
 
 import javax.inject.Inject;
 import java.io.PrintWriter;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.Objects;
 
-@CommandLine.Command(name = "shell", aliases = "sh",
-        versionProvider = MqttCLIMain.CLIVersionProvider.class,
+@CommandLine.Command(name = "shell", aliases = "sh", versionProvider = MqttCLIMain.CLIVersionProvider.class,
         description = "Starts MqttCLI in shell mode, to enable interactive mode with further sub commands.",
-        footer = {"", "@|bold Press Ctl-C to exit.|@"},
-        synopsisHeading = "%n@|bold Usage|@:  ",
-        descriptionHeading = "%n",
-        optionListHeading = "%n@|bold Options|@:%n",
-        commandListHeading = "%n@|bold Commands|@:%n",
-        separator = " ")
-
+        footer = {"", "@|bold Press Ctl-C to exit.|@"}, synopsisHeading = "%n@|bold Usage|@:  ",
+        descriptionHeading = "%n", optionListHeading = "%n@|bold Options|@:%n",
+        commandListHeading = "%n@|bold Commands|@:%n", separator = " ")
 public class ShellCommand implements Runnable {
 
-    private static final String DEFAULT_PROMPT = "mqtt> ";
-    private static String prompt = DEFAULT_PROMPT;
+    private static final @NotNull String DEFAULT_PROMPT = "mqtt> ";
+    private static @NotNull String prompt = DEFAULT_PROMPT;
 
+    //TODO: This is never set
```

Comment:
whats that about?

Reply:
This looks like a bug. I left the TODO in order to mark it for later tickets as this pull request is already big enough.